### ISSUE TITLE
Idea/Feature: Region3 Based F3X

### ISSUE THREAD

**Issue by NNickey** [F3XTeam/ROBLOX-Building-Tools#71]
I have tried to make a Region3 based F3X tool where if you tried to extend any selected parts outside the Region3, it would stop on how far it can go before extending past the Region3 walls. So far, I couldn't find a function to tell if a corner of a part went outside the Region3.
People kept saying, make 6 Region3's around the area.
The solution does work, but then it's slower at responding when I want to move it.

tl;dr, Make it so if the F3X tool was given a Region3, restrict it to only being able to build in that area.

**Comment by GigsD4X** [comment]
Same as #53 :)
